### PR TITLE
feat: add docs on this repository purpose and usage

### PR DESCRIPTION
**Pull request by gforcada**
Maybe we want to move that somewhere else, but meanwhile, better documented here than nowhere 🙈

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,61 @@
...
+python3.11 -m venv venv
+. venv/bin/activate
+pip install tox
+tox -e test
+```
+
+Note: the GitHub Actions workflow is __not__ installed on each repository,
+but rather shared by all repositories.
+
+The workflow definition can be found in this repository in `shared-workflows/tests.yml`.
```
Comment (mauritsvanrees) [inline README.md:30]: Can you link to the organisation settings page where we need to enable the workflow for a repository?
I always have trouble finding it.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,61 @@
...
+
+```shell
+tox -e format
+tox -e lint
+```
+
+Note: they are based on `pre-commit`.
+Each repository needs to be enabled on [pre-commit.ci](https://pre-commit.ci/)
+for the formatting/linting to run on each commit.
+
```
Comment (mauritsvanrees) [inline README.md:45]: Maybe add a few paragraphs:

Note that pre-commit is smart: it only checks the files that you touch.

Not everyone like pre-commit hooks, and this is fine. We have configuration, but no hooks are installed by default. If you _do_ want to automatically run pre-commit when you make a commit, you can:

```
pre-commit install --install-hooks
```

**Review by mauritsvanrees — APPROVED** [review]
LGTM. Two suggestions in comments.

File: README.md (hunk 1 above)
Comment (gforcada) [reply README.md:30]: Good idea 👍🏾

File: README.md (hunk 2 above)
Comment (gforcada) [reply README.md:49]: I made it a bit shorter and to the point, but I hope is enough 😄 🍀

**Review by mauritsvanrees — APPROVED** [review]

**Pull request merged**